node-syscall/syscall.cc

```diff
@@ -25,7 +25,17 @@ intptr_t toNative(Local<Value> value) {
     Local<Array> array = Local<Array>::Cast(value);
     intptr_t* native = reinterpret_cast<intptr_t*>(malloc(array->Length() * sizeof(intptr_t))); // TODO memory leak
     for (uint32_t i = 0; i < array->Length(); i++) {
+#if (NODE_MODULE_VERSION >= NODE_6_0_MODULE_VERSION)
```

Comment:
What's the difference between `NODE_MODULE_VERSION` (here) and `NODE_MAJOR_VERSION` (above, on line 10)? Is it better to use `NODE_MODULE_VERSION` here?

Reply:
I honestly don't know; rather pattern matching when it comes to C++... I've switched to using `NODE_MAJOR_VERSION`